Campus Density/FeedbackQuestions/WaitTimeQuestion.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  WaitTimeQuestion.swift
+//  Campus Density
+//
+//  Created by Changyuan Lin on 12/12/20.
+//  Copyright © 2020 Cornell DTI. All rights reserved.
+//
+
+import UIKit
+
+protocol WaitTimeQuestionDelegate: class {
+    func waitTimeWasChanged(waitTime: Int)
+}
+
+class WaitTimeQuestion: FeedbackQuestion, UIPickerViewDataSource, UIPickerViewDelegate {
+    weak var delegate: WaitTimeQuestionDelegate?
+    let options = ["0-2", "2-4", "4-6", "6-8", "8-10", "10-12", "12-14", "14-16", "16-18", "18-20", "20+"]
```

Comment:
I like how you added more granularity to the waittime options I should do that too. However, is it necessary to include `"16-18", "18-20", "20+"`?

Reply:
Not sure, I think we just chose buckets of two minutes. I agree that those buckets on the high end seem less meaningful as 14 and 16 minutes don't feel as different as 2 and 4 minutes. We should probably standardize them later so the data coming into the backend has the same source options.